.github/workflows/pull-request.yml

```diff
@@ -30,28 +30,3 @@ jobs:
 
       - name: Run Tests
         run: pnpm test
-
-  code-quality:
```

Comment:
This is not a part of my change in PR #13. Is it intended to remove them all?

Reply:
I'm thinking the test step above runs the required checks, but could be wrong.

Reply:
@kai-nguyen-aligent This one was added later during the Github migration because the formatting checks weren't running. Now that they're running, we don't need the extra step any more.